network/topics/params/peer_score.go

```diff
@@ -45,13 +46,11 @@ func PeerScoreThresholds() *pubsub.PeerScoreThresholds {
 }
 
 // PeerScoreParams returns peer score params according to the given options
-func PeerScoreParams(oneEpoch, msgIDCacheTTL time.Duration, disableColocation bool, ipWhilelist ...*net.IPNet) *pubsub.PeerScoreParams {
-	if oneEpoch == 0 {
-		oneEpoch = oneEpochDuration
-	}
+func PeerScoreParams(netCfg networkconfig.NetworkConfig, msgIDCacheTTL time.Duration, disableColocation bool, ipWhilelist ...*net.IPNet) *pubsub.PeerScoreParams {
+	epochDuration := netCfg.EpochDuration()
 
 	// P7 calculation
-	behaviourPenaltyDecay := scoreDecay(oneEpoch*10, decayInterval)
```

Comment:
doesn't seem exactly like before?

Reply:
decayInterval was one epoch:
```go
decayInterval = 32 * (time.Second * 12) // One epoch
```

Reply:
@MatheusFranco99 can you please confirm this is correct?

Reply:
I see now 👍